### PR TITLE
Fix multi match look in rp contrib

### PR DESCRIPTION
**Pull request by bradleymarques**

### CODE REVIEW

File: evennia/contrib/rpg/rpsystem/tests.py [hunk 1]
```diff
@@ -2,11 +2,8 @@
 Tests for RP system
 
 """
-import time
-
-from anything import Anything
```
Comment (bradleymarques) [inline evennia/contrib/rpg/rpsystem/tests.py:7]: These were unused imports.

File: evennia/utils/dbserialize.py [hunk 2]
```diff
@@ -502,7 +502,6 @@ def deserialize(obj):
     """
 
     def _iter(obj):
-        # breakpoint()
```
Comment (bradleymarques) [inline evennia/utils/dbserialize.py:505]: 🧹

File: evennia/contrib/rpg/rpsystem/tests.py [hunk 3]
```diff
@@ -338,6 +334,46 @@ def setUp(self):
         self.char1.swap_typeclass(rpsystem.ContribRPCharacter)
         self.char2.swap_typeclass(rpsystem.ContribRPCharacter)
 
+    def test_REMOVE_ME_PLEASE_DEAR_GOD_REMOVE_ME(self):
```
Comment (bradleymarques) [inline evennia/contrib/rpg/rpsystem/tests.py:337]: **TODO: REMOVE THIS**

Just added so I could trigger the "normal" `search` method in tests for debugging.

**Comment by Griatch** [comment]
Ping. If this is being worked on, it's fine, but please elaborate in the description of this PR exactly what the problem is, if it tries to close a particular ticket etc.

**Comment by bradleymarques** [comment]
Sorry @Griatch - lost steam on this. I'll rather open it as an Issue describing the bug.

**Pull request closed without merging**